### ISSUE TITLE
`expect.sh` no such file or directory

### ISSUE THREAD

**Issue by Varstahl** [babagreensheep/jellyfin-tizen-docker#2]
I was puzzled for a hot minute until I realised it was a problem with cloning, posting here in case someone else gets stuck.

The error:
```
 => ERROR [28/28] RUN ./expect.sh                                   0.4s
------
 > [28/28] RUN ./expect.sh:
": no such file or directory "
------
executor failed running [/bin/sh -c ./expect.sh]: exit code: 1
```

Solution: when cloning make sure that the lines in the expect.sh script are LF terminated, and not CRLF terminated.

**Comment by babagreensheep** [comment]
Sorry for my ignorance - how should I fix this?

**Comment by Varstahl** [comment]
Oh, sorry for the confusion, everything works in the repository. If you use the `git clone` command, depending on some settings on the PC you're issuing it from, carriage return characters can get injected into the file.

If you want you can just mention in the readme that if they're having problems building the image with errors related to `expect.sh` in the last step, they should download [https://github.com/babagreensheep/jellyfin-tizen-docker/blob/master/expect.sh](https://github.com/babagreensheep/jellyfin-tizen-docker/blob/master/expect.sh) directly.

Btw, thanks a lot for making this up, saved me a ton of time.

**TL;DR**: I was being stupid, and thought I'd mention it for others who might have the same problem building it on a Windows machine.

**Comment by babagreensheep** [comment]
Oh that's really interesting bug I'll fire up a windows vm one day and see if I can clean it up. 

You're welcome! Hope you managed to get it up on your tv.

**Comment by Varstahl** [comment]
Yup, worked fine, I'd put the `sdb devices` earlier in the readme though: I was *this* close to going downstairs checking the code of the TV. Luckly I thought "wait, there *must* me a way to list devices connected", and then I read the comment after the command :D

**Comment by Varstahl** [comment]
Speaking of which I'll submit a PR for the Readme with a few information and some cleanup to make the process smoother while I'm at it, should be easy enough.
